### PR TITLE
Fix the typo regarding the name of the method called assign_voltage_s…

### LINKED ISSUE #342: Rename `assig_voltage_source_to_sheet` to `assign_voltage_source_to_sheet`
@MaxJPRey, please rename `assig_voltage_source_to_sheet` to `assign_voltage_source_to_sheet`:
https://github.com/pyansys/PyAEDT/blob/130ecc7f80c648a7d3eb2606e8c9d8ce9e1b89ed/pyaedt/hfss.py#L1712

Additionally, create a separate PR with an `analyse --> analyze` refactor.

_Originally posted by @akaszynski in https://github.com/pyansys/PyAEDT/issues/333#issuecomment-885887008_

### PR DESCRIPTION
**Pull request by MaxJPRey**
Fix #342 
Correct the typo in the name of the method `assign_voltage_source_to_sheet`

### CODE REVIEW

**Comment by jleonatti** [comment]
I think the deprecation best practice should be followed here as well. It is probably best to leave the typo API as is and link it to a new method with the correct name, while adding a deprecation warning to the old API. @maxcapodi78 agree?

**Comment by MaxJPRey** [comment]
@jleonatti @maxcapodi78 I will follow the deprecation best practice here as well.

File: pyaedt/hfss.py [hunk 1]
```diff
@@ -1713,6 +1714,19 @@ def create_lumped_port_to_sheet(self, sheet_name, axisdir=0, impedance=50, portn
     def assig_voltage_source_to_sheet(self, sheet_name, axisdir=0, sourcename=None):
         """Create a voltage source taking one sheet.
 
+        .. deprecated:: 0.4.0
+           Use :func:`Hfss.assign_voltage_source_to_sheet` instead.
+
+        """
+
+        warnings.warn('`assig_voltage_source_to_sheet is deprecated`. Use `assign_voltage_source_to_sheet` instead',
```
Comment (jleonatti) [inline pyaedt/hfss.py:1722]:
```suggestion
        warnings.warn('`assig_voltage_source_to_sheet is deprecated`. Use `assign_voltage_source_to_sheet` instead.',
```
Small suggestion.

**Review by jleonatti — APPROVED** [review]

**Review by akaszynski — APPROVED** [review]

**Comment by akaszynski** [comment]
Thanks @jleonatti for ensuring we're following depreciation guidelines.

**Pull request merged**